CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to syrinscape-control
+
+
+Code and content contributions are accepted. Please feel free to submit issues to the issue tracker or submit merge requests for code/content changes. Approval for such requests involves code and (if necessary) design review by the Maintainers of this repo.
+
+Please ensure there is an open issue about whatever contribution you are submitting. Please also ensure your contribution does not duplicate an existing one.
+
+## Developer Tooling
+
+To start, clone this repository and either place it in or symlink it to your `Data/modules/syrinscape-control` user data directory.
+
+To provide type and i18n support, this repository uses a postinstall script that symlinks your local Foundry installation. For this to work, copy `example-foundry-config.yaml` and rename it to `foundry-config.yaml`, then replace the value of the `installPath` field.
+
+Once this is done you can run `npm install` to install all relevant dependencies. This includes `eslint`, which provides formatting support.
+
+For vscode, you will need to create a `.vscode/settings.json` file with the following:
+
+```json
+{
+  "eslint.enable": true,
+  "eslint.validate": ["javascript", "handlebars", "html"]
+}
+```
+
+Also copy the following into your `.vscode/settings.json` to support i18n-ally:
+```json
+"i18n-ally.localesPaths": [
+  "foundry/lang",
+  "lang"
+],
+"i18n-ally.keystyle": "nested",
+```
+
+### VSCode support for i18n
+
+If you are using VSCode, the i18n Ally (ID: `lokalise.i18n-ally`) extension will preview the content of i18n strings by pulling from both `lang/en.json` as well as the symlinked core translation files at `foundry/lang/en.json`.
+
+## Code
+
+Here are some guidelines for contributing code to this project.
+
+To contribute code, [fork this project](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and submit a [pull request (PR)](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request) against the correct development branch.
+
+### Style
+
+Please attempt to follow code style present throughout the project. An ESLint profile is included to help with maintaining a consistent code style. All warnings presented by the linter should be resolved before an PR is submitted.
+
+- `npm run lint` - Run the linter and display any issues found.
+- `npm run lint:fix` - Automatically fix any code style issues that can be fixed.
+
+### Linked Issues
+
+Before (or alongside) submitting an PR, we ask that you open a feature request issue. This will let us discuss the approach and prioritization of the proposed change.
```

Comment:
```suggestion
Before (or alongside) submitting a PR, we ask that you open a feature request issue. This will let us discuss the approach and prioritization of the proposed change.
```